src/games/rcll/production.clp

```diff
@@ -254,7 +254,7 @@
 			 else
 				(if (not (any-factp ((?exp exploration-report)) (and (eq ?exp:name ?mname)
 				                                                     (eq ?exp:rtype RECORD)
-				                                                     ?exp:correctly-reported)))
```

Comment:
do we assert correctly reported after exploration?

Reply:
yes, through this rule 
```
(defrule exploration-set-ground-truth
	(machine (name ?n) (team ?team) (rotation ?rotation) (zone ?zone))
	(not (exploration-report (name ?n) (rtype RECORD) (correctly-reported TRUE)))
	(gamestate (phase ?phase))
	(send-mps-positions (phases $?phases&:(member$ ?phase ?phases)))
	=>
	(printout t "Exploration Phase over, sending ground truth of machines" crlf)
	(do-for-all-facts ((?exp exploration-report))
		(retract ?exp)
	)
	(do-for-all-facts ((?m machine))
		(assert (exploration-report (name ?m:name) (rtype RECORD) (zone ?zone)
		                            (rotation ?rotation) (correctly-reported TRUE)))
	)
)
```